src/context/mod.rs

```diff
@@ -318,6 +320,11 @@ pub trait Context<'a>: ContextInternal<'a> {
         JsBuffer::new(self, size)
     }
 
+    #[cfg(feature = "napi-runtime")]
+    fn date<V: Into<f64>>(&mut self, date: V) -> Handle<'a, JsDate> {
```

Comment:
Oh good catch, this is missing from the RFC. I added it.

I'd call the `date` parameter here `value` as well.